kafka.clients/pom.xml

```diff
@@ -0,0 +1,69 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>dev.galasa</groupId>
+		<artifactId>galasa-wrapping-parent</artifactId>
+		<version>0.33.0</version>
+	</parent>
+
+	<artifactId>kafka.clients</artifactId>
+	<version>3.7.0</version>
+	<packaging>bundle</packaging>
+
+	<name>Galasa wrapped version of the kafka-client package</name>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.apache.kafka</groupId>
+			<artifactId>kafka-clients</artifactId>
+			<version>3.7.0</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.kafka</groupId>
+			<artifactId>kafka-server-common</artifactId>
+			<version>3.7.0</version>
+		</dependency>
+		<dependency>
```

Comment:
Might be better to wrap this individually ? Other things could usefully make use of slf4j. The default implementation within this basic API jar is pretty crappy, but at least it logs to stdout. A better thing is to configure it to use log4j or similar, but you'd need to wrap an extra dependency for that I guess.

slf4j is the way to go for all logging in my opinion, but we don't use it internally anywhere yet (shame)... maybe if it was available we'd start using it more...

Reply:
or perhaps I'm not understanding this.

Reply:
Hi Mike. slf4j is a dependency of the Kafka libraries hence why it needs to be added as a dependency here and as an Import-Package of the bundle. Without slf4j-api as a dependency here, and wrapping it in it's own OSGi wrapper, leads to this error: `Caused by: java.lang.ClassNotFoundException: org.slf4j.LoggerFactory not found by kafka.clients`. So it's needed here to allow our dev.galasa:kafka.clients bundle to find the Logger class at runtime.

Reply:
Ok. Great